schoolopy/main.py

```diff
@@ -11,18 +11,18 @@
 
 
 class Schoology:
-    _ROOT = 'https://api.schoology.com/v1/'
     key = ''
     secret = ''
     limit = 20
     start = 0
 
-    def __init__(self, schoology_auth):
+    def __init__(self, schoology_auth, api_server='https://api.schoology.com/v1'):
         if not schoology_auth.authorized:
             raise AuthorizationError('Auth instance not authorized. Run authorize() after requesting authorization.')
         self.key = schoology_auth.consumer_key
         self.secret = schoology_auth.consumer_secret
         self.schoology_auth = schoology_auth
+        self.api_server = api_server + '/'
```

Comment:
I would prefer to have the / included in the parameter.

Reply:
@AndrewLester wdyt?

Reply:
So should I get rid of ` + '/'`?

Reply:
Fixed in https://github.com/gadhagod/schoolopy/commit/46940ca43f68dff35f7de047dea6735a0ae55309.